### ISSUE TITLE
Importer crashes when loading study_es_0 with gene panel data

### ISSUE THREAD

**Issue by sandertan** [cBioPortal/cbioportal#2683]
I was testing loading sample `study_es_0` with the new additions of gene panels and gene sets. 

Updates in #2506 regarding gene panels:
- Perl wrapper to import gene panel data
- Updated documentation on Import-Gene-Panels.md and Loading-Sample-Cancer-Study.md
- Add error message to Importer when used has not imported gene panes before importing study gene panel data.

Still I run into an error when loading gene panel data. Updated documentation to prepare before loading loading `study_es_0` in here: [#2506 Loading-Sample-Cancer-Study.md](https://github.com/thehyve/cbioportal/blob/2b1fe01e669e55f9b11dba2c0881330af7a7b562/docs/Load-Sample-Cancer-Study.md#import-gene-panels-and-gene-sets-for-sample-study)

```
Reading data from:  /Users/sander/Desktop/study_es_0_panel/data_gene_panel_matrix.txt

ABORTED!
java.lang.RuntimeException: Cannot find genetic profile study_es_0_mutations in the database.
at org.mskcc.cbio.portal.scripts.ImportGenePanelProfileMap.getProfileIds(ImportGenePanelProfileMap.java:178)
at org.mskcc.cbio.portal.scripts.ImportGenePanelProfileMap.importData(ImportGenePanelProfileMap.java:119)
at org.mskcc.cbio.portal.scripts.ImportGenePanelProfileMap.run(ImportGenePanelProfileMap.java:97)
at org.mskcc.cbio.portal.scripts.ConsoleRunnable.runInConsole(ConsoleRunnable.java:145)
at org.mskcc.cbio.portal.scripts.ImportGenePanelProfileMap.main(ImportGenePanelProfileMap.java:206)
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
[91mError occurred during data loading step. Please fix the problem and run this again to make sure study is completely loaded.[0m
Traceback (most recent call last):
  File "/Users/sander/Data/cbioportal/core/src/main/scripts/importer/metaImport.py", line 118, in <module>
    cbioportalImporter.main(args)
  File "/Users/sander/Data/cbioportal/core/src/main/scripts/importer/cbioportalImporter.py", line 424, in main
    process_directory(jvm_args, study_directory)
  File "/Users/sander/Data/cbioportal/core/src/main/scripts/importer/cbioportalImporter.py", line 322, in process_directory
    import_study_data(jvm_args, meta_filename, data_filename, study_meta_dictionary[meta_filename])
  File "/Users/sander/Data/cbioportal/core/src/main/scripts/importer/cbioportalImporter.py", line 131, in import_study_data
    run_java(*args)
  File "/Users/sander/Data/cbioportal/core/src/main/scripts/importer/cbioportal_common.py", line 796, in run_java
    raise RuntimeError('Aborting due to error while executing step.')
RuntimeError: Aborting due to error while executing step.
```

**Comment by pieterlukasse** [comment]
The reason is that `DaoCancerStudy` (and its `recacheAll` initialization) is called too late in the process, meaning `DaoGeneticProfile`'s cache (`DaoGeneticProfile.byStableId`) is still empty when `ImportGenePanelProfileMap.getProfileIds()` is called. Solution is to have a static initialization of the cache in `DaoGeneticProfile`, independent of `DaoCancerStudy`'s init, just like is done in `DaoCancerStudy`.

**Comment by sandertan** [comment]
Fixed in #2506
